test/test_queries.py

```diff
@@ -21,7 +21,7 @@
 @pytest.fixture
 def config():
     config = Configuration()
-    config.fmt = 'json'
+    config.fmt = 'table'
```

Comment:
I'm a bit confused about how this isn't breaking the other `query_tests`. I think their expected output is still in JSON and wouldn't this make the result be the table format?